solr/modules/cross-dc/build.gradle

```diff
@@ -0,0 +1,70 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Cross-DC Replication Plugins'
+
+ext {
+  // Use of the security manager is not supported on windows.
```

Comment:
Why is this concern specific to cross-dc?  We use Mockito throughout Solr tests.

Reply:
It's because of a feature of mockito that spins up new jvms I think?
We spent a ton of time trying to get it to work, to no avail.
Since the security manager is going away anyways, it's not the worst thing.